src/parser/core/modules/DoTs.ts

```diff
@@ -7,15 +7,12 @@ import {Data} from 'parser/core/modules/Data'
 import {Invulnerability} from 'parser/core/modules/Invulnerability'
 import {Statuses} from 'parser/core/modules/Statuses'
 
-const SECONDS_PER_MINUTE = 60
+const MILLISECONDS_PER_MINUTE = 60000
 
-/** Mapping of a status ID to the duration it was up. */
-export interface DotDurations {
-	[statusID: number]: number
-}
-
-interface DotTargets {
-	[targetID: string]: DotDurations
+type DotTracking = Map<number, Map<string, DotTargetTracking>>
```

Comment:
Should this not be more explicit to `Status['id']` and `Actor['id']` or something?

Reply:
Wouldn't block on it, but yes - preference semantic lookup types like `Status['id']` over straight `number` or similar, as while they're _equivalent_, it'll help keep them up to date as types evolve